app/bundles/CoreBundle/Templating/Helper/SlotsHelper.php

```diff
@@ -62,7 +62,7 @@ public function hasContent($names)
             foreach ($names as $n) {
                 // strip tags used to ensure we don't have empty tags.
                 // Caused a bug with hasContent returning incorrectly. Whitelisted img to fix
-                $hasContent = (bool) strip_tags(trim($this->slots[$n]), '<img><iframe>');
+                $hasContent = (bool) strip_tags(trim($this->slots[$n]), '<img alt="Whitelisted image"><iframe>');
```

Comment:
```suggestion
                $hasContent = (bool) strip_tags(trim($this->slots[$n]), '<img alt="Allowed image"><iframe>');
```
Let's use 'allowed' or something similar rather than 'whitelisted' please.  For reference: https://developers.google.com/style/inclusive-documentation?hl=en